bin/configure-tower-projects.py

```diff
@@ -514,16 +503,60 @@ def create_compute_environment(self) -> str:
                         "ec2KeyPair": None,
                         "imageId": None,
                         "securityGroups": [],
-                        "ebsBlockSize": None,
+                        "ebsBlockSize": 250,
```

Comment:
Arbitrary number?

Reply:
The default is 50 GB, and we're having issues with running out of disk space. I wanted to add some buffer, and the largest files that I'm aware of are images from HTAN, which can be as large as 190 GB. I rounded off the number to 250 GB. We might need to increase this if we continue to see disk space issues.